app/src/main/java/com/locoquest/app/dao/IBenchmarkDAO.kt

```diff
@@ -39,4 +41,6 @@ fun main() {
             println("Error: ${t.message}")
         }
     })
-}
```

Comment:
I tried to make a modification to this code so that retrofit building happens only on the retrofitclientinstance.KT. Not a full implementation, but it looks like you might be able to get one of your retrofit builders?

Reply:
The factory/singleton should be in RetrofitClientInstance.
The call operations should be in a Service.
While you're here, replace the println statements with proper logging.
- Logging has the concept of logging levels.  They can be filtered and aggregated based on those levels.
- No one will see printStackTrace.